pulsar-broker/src/main/java/org/apache/pulsar/broker/resourcegroup/ResourceQuotaCalculatorImpl.java

```diff
@@ -80,6 +74,12 @@ public long computeLocalQuota(long confUsage, long myUsage, long[] allUsages) th
         float myUsageFraction = (float) myUsage / totalUsage;
         float calculatedQuota = max(myUsage + residual * myUsageFraction, 0);
 
+        // If the calculated rate ends up being 0, set it to 0. rate value of 0 or less is assumed to mean that
```

Comment:
should the comment say "set it to 1"

Reply:
It isn't clear why 0 is assumed to mean "no rate limiter is set", if negative values also connote the same thing. It is conceivable that a rate limiter could be set, and the rate would be set to 0, right?

Reply:
thank you @ravi-vaidyanathan , will fix the comment.

Reply:
@kaushik-develop else-where in the RateLimiter code, that is the convention that is currently followed. See the code here:

https://github.com/apache/pulsar/blob/a7bdc5e25778ada8c781e9b3f9cc4694e5fd3f58/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/PrecisPublishLimiter.java#L97

And I don't plan to change the precedence (as that will only cause more confusion).